Cargo.toml

```diff
@@ -33,9 +33,9 @@ serde_derive = { version = "1.0", optional = true }
 unicase_serde = { version = "0.1.0", optional = true }
 
 [dev-dependencies]
-hyper = "0.10"
 serde_json = "1.0"
 serde_test = "1.0"
+tokio = "0.2"
```

Comment:
We shouldn't need this as a dev dependency because of the macros that Rocket provides for starting up an async main function.

Reply:
After replacing the macros, I removed the dependency.